test/unit/lexer/diagnostic_reporter.d

```diff
@@ -51,25 +51,6 @@ unittest
     assert(reporter.warningCount == 1);
 }
 
-@("deprecations: hex string literal")
```

Comment:
maybe replace this with sth. else that's deprecated (like e.g. complex numbers), s.t. the deprecation report test is kept?

Reply:
For reference, it's unlikely that the deprecation of complex numbers will end anytime soon: https://github.com/dlang/dmd/pull/7640

Reply:
The depreciation for complex numbers hasn't even been started yet, unfortunately.  So, I chose C-style array pointers for now.  I'll probably have to find a new one in a couple of months though.

Reply:
Well, that didn't work either, I think because the depreciation isn't emitted in the lexer.  Trying invalid integers.